source/classes.tex

```diff
@@ -513,6 +513,8 @@
 if it is
 \begin{itemize}
 \item a friend declaration\iref{class.friend},
+\item a \grammarterm{deduction-guide}\iref{temp.deduct.guide},
+\item a \grammarterm{template-declaration} whose \grammarterm{declaration} is one of the above,
```

Comment:
I agree that a _deduction-guide_ does not declare a new member (as explained in the commit comment), but here materializes another rule that isn't mentioned. (Plus, it seems to me a template friend is still a friend declaration of sorts.)

Reply:
> Please don't mix unrelated changes in the same commit.

I wouldn't say that it's entirely unrelated, so should I just change the commit message & PR name?

> It seems to me a template friend is still a friend declaration of sorts.

I consider a friend template to be a friend declaration as much as I consider a class template to be a class declaration.

> here materializes another rule that isn't mentioned

I'm assuming that you're referring to *deduction-guides* declared outside of a *template-declaration*. With respect to such declarations, I don't see any wording (normative or informative) that explicitly states such *deduction-guides* can be declared, however we can deduce that it's allowed from: 
- grammar (a *deduction-guide* (without a *template-head*) is a *declaration* as well as a *member-declaration*) and,
- normative wording ([[over.match.class.deduct] p1.4](http://eel.is/c++draft/over.match.class.deduct#1.4) says "if any" when referring to the template parameters; you can't explicitly specialize a deduction guide so the only context where this would occur is when there is no *template-head*)